packages/vscode-extension/src/utils/commonUtils.ts

```diff
@@ -134,3 +137,16 @@ export function anonymizeFilePaths(stack?: string): string {
 
   return updatedStack;
 }
+
+export async function isTeamsfx(): Promise<boolean> {
```

Comment:
Looks `debug/commonUtils` already has similar method `isFxProject`, can we reuse it?

Reply:
```debug/commonUtils``` needs path as a parameter. If I use this method, also need to add a method to handle path logic.